### PR TITLE
Fixes #1933  text and logical definition club cell

### LINKED ISSUE #1933: [Logical def] Club cell
**CL term**
'Club cell'  CL:0000158

**Suggested revision of logical definition**
`'bronchial epithelial cell'
 and ('has characteristic' some domed)
 and ('has plasma membrane part' some microvillus)
 and ('capable of' some 'secretion by cell')
 and ('capable of' some 'surfactant homeostasis')`

Club cells are found in bronchioles, not in bronchi. Suggested new logical definition:
`'secretory cell'
 and ('part of' some 'epithelium of bronchiole')
 and ('has characteristic' some domed)
 and ('has plasma membrane part' some microvillus)
 and ('capable of' some 'surfactant homeostasis')`

### PR DESCRIPTION
**Pull request by aleixpuigb**
Fixes #1933  logical definition club cell

### CODE REVIEW

File: src/ontology/cl-edit.owl [hunk 1]
```diff
@@ -4843,7 +4843,7 @@ AnnotationAssertion(oboInOwl:hasExactSynonym obo:CL_0000158 "Clara cell")
 AnnotationAssertion(oboInOwl:hasExactSynonym obo:CL_0000158 "bronchiolar non-ciliated cell")
 AnnotationAssertion(oboInOwl:hasExactSynonym obo:CL_0000158 "club cell of bronchiole")
 AnnotationAssertion(rdfs:label obo:CL_0000158 "club cell")
-EquivalentClasses(obo:CL_0000158 ObjectIntersectionOf(obo:CL_0002328 ObjectSomeValuesFrom(obo:RO_0000053 obo:PATO_0001789) ObjectSomeValuesFrom(obo:RO_0002104 obo:GO_0005902) ObjectSomeValuesFrom(obo:RO_0002215 obo:GO_0032940) ObjectSomeValuesFrom(obo:RO_0002215 obo:GO_0043129)))
+EquivalentClasses(obo:CL_0000158 ObjectIntersectionOf(obo:CL_0000151 ObjectSomeValuesFrom(obo:BFO_0000050 obo:UBERON_0002051) ObjectSomeValuesFrom(obo:RO_0000053 obo:PATO_0001789) ObjectSomeValuesFrom(obo:RO_0002104 obo:GO_0005902) ObjectSomeValuesFrom(obo:RO_0002215 obo:GO_0043129)))
```
Comment (bvarner-ebi) [inline src/ontology/cl-edit.owl:4846]: This cell type has been noted in various locations along the tracheobronchial tree. 
See https://www.ncbi.nlm.nih.gov/pmc/articles/PMC7528681/figure/F2/

Consider converting the equivalence axiom into subClass axioms.

Comment (aleixpuigb) [reply src/ontology/cl-edit.owl:4846]: It looks like it varies [depending on the species](https://www.ncbi.nlm.nih.gov/pmc/articles/PMC6290947/):
image

**Review by bvarner-ebi — COMMENTED** [review]
- The text definition should be aligned with changes made to the axioms in this PR. 
For example, if the axiom regarding "dome-shaped" morphology is removed, an analogous change should be made to the text def. If the location was changed to tracheobronchial tree, that should also be reflected in the text definition, which presently only references bronchioles.

- The first sentence of the text definition is missing differentia and is not aligned with axioms, i.e., no axiom is present regarding it being a progenitor cell.

- The exact synonyms are also not aligned with the changes made in this PR. Since this class has been edited to represent this cell type anywhere along the tracheobronchial tree, "club cell of bronchiole" would be either a subclass or narrow synonym, not exact.

- "bronchiolar non-ciliated cell" seems overly narrow (bronchiolar) and overly broad (non-ciliated cell) at the same time and also has no dbxref, so consider revising or removing. It is likely present due to the mapping to FMA:14119.

- There is mention of other "club cells" in other organs in the comments. Will there be a parent class for all these "club cells"? With that in mind, consider labelling this as 'tracheobronchial club cell' or 'lung club cell' instead of just 'club cell'.

**Comment by bvarner-ebi** [comment]
> > One solution would be requesting a NTR 'surfactant secretion' to GO. I would like to maintain `capable_of some surfactantant homeostasis` for CC and I think the logical definition of `surfactant secreting cell` is too broad and should be moved to Subclasses as well. However, I will do a bit more research on the surfactant secretion (That will help with the NTR for GO, as well).
> 
> [Ticket created in GO](https://github.com/geneontology/go-ontology/issues/25317).

To note, `surfactant secreting cell` is still inferred.

**Comment by aleixpuigb** [comment]
> - The text definition should be aligned with changes made to the axioms in this PR. 
For example, if the axiom regarding "dome-shaped" morphology is removed, an analogous change should be made to the text def. If the location was changed to tracheobronchial tree, that should also be reflected in the text definition, which presently only references bronchioles.
> - The first sentence of the text definition is missing differentia and is not aligned with axioms, i.e., no axiom is present regarding it being a progenitor cell.

Agreed, I have changed it:
          `A non-mucous, epithelial secretory cell that is part of the tracheobronchial tree. A club cells has short microvilli but no cilia. A club cell is able to multiply and differentiate into ciliated cells to regenerate the bronchiolar epithelium and it also protects the tracheobronchial epithelium.`

> - The exact synonyms are also not aligned with the changes made in this PR. Since this class has been edited to represent this cell type anywhere along the tracheobronchial tree, "club cell of bronchiole" would be either a subclass or narrow synonym, not exact.
> - "bronchiolar non-ciliated cell" seems overly narrow (bronchiolar) and overly broad (non-ciliated cell) at the same time and also has no dbxref, so consider revising or removing. It is likely present due to the mapping to FMA:14119.

I would remove both synonyms.


> - There is mention of other "club cells" in other organs in the comments. Will there be a parent class for all these "club cells"? With that in mind, consider labelling this as 'tracheobronchial club cell' or 'lung club cell' instead of just 'club cell'.

[For prostate club cells:](https://www.cell.com/cell-reports/pdf/S2211-1247(18)31877-1.pdf)
`Prostate SCGB1A1+ cells are similar in morphology and transcriptomic profile to Clara, or club, cells [...]. Prostate club cells are similar to lung club cells in their enrichment of immunomodulatory programs (Figure 4; Table S4), but their function in the prostate or prostatic urethra has not been tested.`
This cell is missing in the ontology. There are three options:
   1. Create a new term club-like cell similar to `club-like cell of the urethral epithelium`.
   2. Create a parental term club cell, and change the label of this term to tracheobronchial club cell.
   3. Expand the text and logical definition to all organs if it is exactly the same cell.

As the function is not known, I am leaning for option 1, but maybe we could bring this issue to the next CL call.

**Comment by bvarner-ebi** [comment]
> A non-mucous, epithelial secretory cell that is part of the tracheobronchial tree. A club cells has short microvilli but no cilia. A club cell is able to multiply and differentiate into ciliated cells to regenerate the bronchiolar epithelium and it also protects the tracheobronchial epithelium.

Thank you for the updated definition.

- "A club cells" --> "A club cell"
- "has short microvilli" - after a cursory review, I did not find this description in the dbxrefs
- "it also protects the tracheobronchial epithelium" - after a cursory review, I did not find this description in the dbxrefs, unless this refers to the regenerative capabilities, in which case it seems redundant.
- As this cell type is described as a progenitor cell in dbxref, consider reflecting that in axioms rather than removing altogether

**Comment by aleixpuigb** [comment]
> > A non-mucous, epithelial secretory cell that is part of the tracheobronchial tree. A club cells has short microvilli but no cilia. A club cell is able to multiply and differentiate into ciliated cells to regenerate the bronchiolar epithelium and it also protects the tracheobronchial epithelium.
> 
> Thank you for the updated definition.
> 
> * "A club cells" --> "A club cell"

Thank you!
> * "has short microvilli" - after a cursory review, I did not find this description in the dbxrefs
> * "it also protects the tracheobronchial epithelium" - after a cursory review, I did not find this description in the dbxrefs, unless this refers to the regenerative capabilities, in which case it seems redundant.

[This reference] provide reference for both points.

> * As this cell type is described as a progenitor cell in dbxref, consider reflecting that in axioms rather than removing altogether
Agreed.

**Pull request merged**